maintainers/maintainer-list.nix

```diff
@@ -3837,6 +3837,11 @@
     githubId = 15692230;
     name = "Muhammad Herdiansyah";
   };
+  konstantsky = {
+    email = "konstantin@konstantsky.pw";
+    github = "konstantinous";
+    name = "Konstantin Khokhlov";
+  };
```

Comment:
should be a separate commit:
```
maintainers: add konstantsky
```
also add githubId

Reply:
Done.